functions.sh

```diff
@@ -627,6 +627,16 @@ if [ -n "$1" ]; then
   FORCE_PASSWORD="$(grep -m1 -e ^FORCE_PASSWORD "$1" |awk '{print $2}')"
```

Comment:
```suggestion
  FORCE_PASSWORD="$(awk '/^FORCE_PASSWORD/ { printf $2; exit }' "$1")"
```

Reply:
Yeah, there are a *lot* of these things throughout the whole codebase.
"FORCE_PASSWORD" seems a bit out of scope for this PR, I have changed the parsing of "DWC_ENABLE" accordingly though. thanks!